code/modules/client/preference/character.dm

```diff
@@ -416,45 +421,45 @@
 	gender = query.item[4]
 	age = text2num(query.item[5])
 	species = query.item[6]
-	language = query.item[7]
```

Comment:
You need to update the select query for this.

Also please append it to the end as opposed to changing all these indexes around.

Reply:
I changed the indexes around to fit in line with the `//character` selects. But I'll append it to the end.